packages/client/src/modules/app/app.tsx

```diff
@@ -54,6 +55,12 @@ export default function App() {
     setCountryInQueryParam(initialCountry);
   }
 
+  useEffect(() =>
+    process.env.NODE_ENV === 'production'
+      ? ReactGA.initialize('UA-160752885-1')
```

Comment:
Please movie this __UA-160752885-1__ into a env variable so that tracking id does not get checked in

Reply:
I have moved it to environment file. Should I share the tracking ID separately to you for updating the production environment?

Reply:
Yup please share it with me privately